### PR TITLE
Extract all localizable strings and use them from a table.

### PR DESCRIPTION
**Pull request by nlutsenko-fb**
cc @grantland @hallucinogen @stanleyw

### CODE REVIEW

**Comment by nlutsenko-fb** [comment]
Silly Travis-CI.
Phew, ok, at least everything is back to normal and ready.

cc @grantland @hallucinogen  :)

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: objective-c
+osx_image: xcode611
```
Comment (hallucinogen) [inline .travis.yml]: we're not using 6.2?

Comment (nlutsenko-fb) [inline .travis.yml]: 6.2 is not available on Travis-CI :(

Comment (nlutsenko-fb) [inline .travis.yml]: They have an issue open -> https://github.com/travis-ci/travis-ci/issues/3373

**Comment by hallucinogen** [comment]
LGTM. Let's do it!

**Pull request merged**